src/main/java/net/geforcemods/securitycraft/blocks/reinforced/BaseReinforcedBlock.java

```diff
@@ -209,6 +224,30 @@ public void randomTick(BlockState state, ServerWorld world, BlockPos pos, Random
 		}
 	}
 
+	@Override
+	@OnlyIn(Dist.CLIENT)
+	public void animateTick(BlockState stateIn, World worldIn, BlockPos pos, Random rand) {
+		if (this == SCContent.REINFORCED_CRYING_OBSIDIAN.get())
+		{
+			if(rand.nextInt(5) == 0)
+			{
+				Direction direction = Direction.func_239631_a_(rand);
+				if(direction != Direction.UP)
+				{
+					BlockPos blockpos = pos.offset(direction);
+					BlockState blockstate = worldIn.getBlockState(blockpos);
+					if(!stateIn.isSolid() || !blockstate.isSolidSide(worldIn, blockpos, direction.getOpposite()))
+					{
+						double d0 = direction.getXOffset() == 0 ? rand.nextDouble() : 0.5D + direction.getXOffset() * 0.6D;
+						double d1 = direction.getYOffset() == 0 ? rand.nextDouble() : 0.5D + direction.getYOffset() * 0.6D;
+						double d2 = direction.getZOffset() == 0 ? rand.nextDouble() : 0.5D + direction.getZOffset() * 0.6D;
```

Comment:
d0 -> xOffset
d1 -> yOffset
d2 -> zOffset

Please also add a new line before the addParticle call.